src/Cache/GlossaryCache.php

```diff
@@ -52,8 +52,8 @@ public function getCache() {
 	 * @return string
 	 */
 	public function getKeyForSubject( DIWikiPage $subject ) {
-		// FIXME Remove wfMemcKey dep.
-		return wfMemcKey( 'ext', 'semanticglossary', $subject->getSerialization() );
+        $cache = self::getCache();
```

Comment:
Looks like that should be `$cache = $this->getCache()`

I'd also avoid introducing the local variable. You can just do `$this->getCache()-->makeKey()`